notebooks/binom_factor_table.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.      print("# AUTOGENERATED from notebooks/binom_factor_table.ipynb", file=f)
</pre><p>I think the linters are complaining as this doesn't have a copyright and fails the black formatter.  Could you try changing this to:</p><p> </p><pre class="ql-syntax" spellcheck="false">print("# Copyright (c) 2023 Graphcore Ltd. All rights reserved.", file=f)
print("# AUTOGENERATED from notebooks/binom_factor_table.ipynb", file=f)
print("# fmt: off", file=f)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore-research/pyscf-ipu/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>